server/app/models/event_stream.rb

```diff
@@ -0,0 +1,54 @@
+module EventStream
+  extend ActiveSupport::Concern
+  CHANNEL = "FirehoseApiEvent".freeze
+
+  included do
+    after_create :publish_create_event
+    after_update :publish_update_event
+    after_destroy :publish_destroy_event
+  end
+
+  def self.channel
+    CHANNEL
+  end
+
+  def publish_create_event    
+    event = {
+      event: 'create',
+      type: self.class.name,
+      object: find_serializer.to_json
```

Comment:
Hmm, comparing this to the RPC serializers in #1873, perhaps it would be better to just use MessagePack serialization for everything on the RPC wire, i.e. this should just be `.to_hash`, not `.to_json`?

Current format is something like hash in json-string in messagepack-string in messagepack-hash?